addons/mass_mailing/models/mailing.py

```diff
@@ -131,12 +131,20 @@ def _get_default_mail_server_id(self):
         default=_get_default_mail_server_id,
         help="Use a specific mail server in priority. Otherwise Odoo relies on the first outgoing mail server available (based on their sequencing) as it does for normal mails.")
     contact_list_ids = fields.Many2many('mailing.list', 'mail_mass_mailing_list_rel', string='Mailing Lists')
-    contact_ab_pc = fields.Integer(string='A/B Testing percentage',
-        help='Percentage of the contacts that will be mailed. Recipients will be taken randomly.', default=100)
-    unique_ab_testing = fields.Boolean(string='Allow A/B Testing', default=False,
+    # A/B Testing
+    ab_testing_description = fields.Html('A/B Testing Description', compute="_compute_ab_testing_description")
```

Comment:
Side question: as everything is based on the campaign, shoudln't those fields be based on group_mass_mailing_campaign directly at model level ? I see some of them are in view, why not here ?